lib/fog/compute/openstack.rb

```diff
@@ -380,6 +381,10 @@ def self.not_found_class
         end
 
         def initialize(options = {})
+          @supported_versions = SUPPORTED_VERSIONS
+          @supported_microversion = SUPPORTED_MICROVERSION
+          @microversion_key = 'X-OpenStack-Nova-API-Version'.freeze
```

Comment:
Why a `freeze` here?

Reply:
This is taken 1-by-1 from the [PR which introduced microversions](https://github.com/fog/fog-openstack/pull/242) for the shared-filesystem-service. We can totally change it, if you prefer.

Reply:
Yeah I remember that one although I realize now that freezing the content of a variable might not make sense. Could you remove the freeze here?

Reply:
sure, we'll do that
\